src/utils/deviceUtils.js

```
@@ -1,76 +1,178 @@
+    return undefined;
+  }
+  return Math.max(1, Math.round(value));
+}
+
+function getNodeRequire() {
+  if (typeof module !== "undefined" && typeof module.require === "function") {
+    return module.require.bind(module);
+  }
+  try {
+    // Using eval avoids bundlers (e.g. Metro) trying to resolve the module at build time.
+    return eval("require");
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)